qiskit_metal/toolbox_metal/exceptions.py

```diff
@@ -66,3 +61,8 @@ class IncorrectQtException(Exception):
     def __init__(self, message: str) -> None:
         prefix = "You should be using be PySide2. \n"
```

Comment:
Grammar of string?
You should be using pySide2.

Reply:
There is an extra "be" in the sentence.

Reply:
fixed